firefox-ios/Client/Frontend/Settings/SyncContentSettingsViewController.swift

```diff
@@ -27,6 +29,14 @@ class ManageFxAccountSetting: Setting {
                 ]
             )
         )
+        
+        notification = NotificationCenter.default.addObserver(
+            forName: .accountLoggedOut,
+            object: nil,
+            queue: .main
+        ) { _ in
+            settings.dismiss(animated: true, completion: nil)
```

Comment:
Generally I don't think we should be strongly retaining the settings VC here in this notification block? These days the overhead from a weak reference is generally not problematic performance-wise and so it's usually safer to avoid retaining the VC if it would otherwise be deallocated. 

```suggestion
        ) { [weak settings] _ in
            settings?.dismiss(animated: true, completion: nil)
```

Reply:
Thanks, done!